src/app/routes/__tests__/expenses.test.tsx

```
@@ -3,18 +3,11 @@ import { render } from '@testing-library/react';
 
 import { createRoutesStub } from 'react-router';
```

ESLint

[error] 'createRoutesStub' is defined but never used.

(@typescript-eslint/no-unused-vars)